components/formats-gpl/src/loci/formats/in/GatanReader.java

```diff
@@ -543,7 +550,6 @@ else if (labelString.equals("DataType")) {
               break;
             case 11:
               core.get(0).pixelType = FormatTools.UINT32;
-              break;
```

Comment:
I'd agree with https://github.com/openmicroscopy/bioformats/pull/1928#issuecomment-132239376 -- best practice is to have the break, because it can introduce subtle bugs if the switch statement is every extended.